independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -157,6 +203,11 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                             invocation.getQuarkusProject().getCodestartResourceLoaders());
             final CodestartProjectDefinition projectDefinition = catalog.createProject(input);
             projectDefinition.generate(invocation.getQuarkusProject().getProjectDirPath());
+
+            if (BuildTool.MAVEN.equals(invocation.getQuarkusProject().getBuildTool()) && parent != null) {
+                addModuleToParentPom(invocation, parent, parentPath);
+            }
```

Comment:
I really don't like this much maven specific behavior in the middle of otherwise common code. I have to think about where a better place to put this is (given the CLI doesn't invoke the maven plugin, it uses this path)..

Reply:
Same as previous comment. Happy to move it if there is a better place.

Reply:
wouldnt the code have failed already at this stage saying parent is not supported for anything but maven ?

Reply:
It would only if the a parent path is passed as param.  lines +116 to +119

Reply:
Sorry, got what you mean now. Leave it just like ? (parent != null) ? @maxandersen

Reply:
I this we can move this in another PR